portfolio/src/main/webapp/script.js

```diff
@@ -444,7 +474,7 @@ async function displayAuth(){
  */ 
 function loginHandler(value){
     document.getElementById("authentication")
-            .innerHTML = '<a href=\"' + value.url + '\">Login</a>'
+            .innerHTML = ` Welcome to the site! <a href="${value.url}">Login Here</a>`;
```

Comment:
this is longer than 80 characters (same with below). Using backticks (`) lets you spread the string among multiple lines:
```
const foo = `bar
  baz
  zoop
  doop`; 
```